src/DynamoCoreWpf/Controls/IncanvasSearchControl.xaml.cs

```diff
@@ -70,6 +81,9 @@ private void ExecuteSearchElement(ListBoxItem listBoxItem)
             var searchElement = listBoxItem.DataContext as NodeSearchElementViewModel;
             if (searchElement != null)
             {
+                Point targetLocation = this.TranslatePoint(new Point(0, 0), WorkspaceView);
+
+                searchElement.Position = targetLocation;
```

Comment:
Could have been this, but just cosmetic :)

`searchElement.Position = TranslatePoint(new Point(), workspaceView);`